SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -387,7 +387,7 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 {
 	InitializeSpatialOutputDevice();
 
-	Dispatcher = NewObject<USpatialDispatcher>();
+	Dispatcher = MakeShared<SpatialDispatcher>();
 	Sender = NewObject<USpatialSender>();
 	Receiver = NewObject<USpatialReceiver>();
```

Comment:
These classes looks like they have the same lifecycle, so it seems to me that cross-referencing between them could be made with raw pointers, and enforce the destruction order for them. Especially since these ones sit in our hot path, it would be better to not have to pin the pointers everytime.

Reply:
I agree

Reply:
Added to summary below.

Reply:
Updated.